bin/proxrox.js

```diff
@@ -41,6 +41,7 @@ program
   .option('--proxy <uri>', 'An IP to proxy via nginx')
   .option('-p --port [port]', 'The port to bind to.', 4000)
   .option('--no-compression', 'Disable GZIP compression', false)
+  .option('--stubStatus', 'Enable stub_status module', false)
```

Comment:
The previous option uses dash-casing instead of camel casing. Could you rename the `stubStatus` option please so that we are consistent? Thanks!